app/app.go

```diff
@@ -25,6 +25,9 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
not sure where this commit is coming from, I created the current branch off of master which [already has these changes]()https://github.com/Kava-Labs/kava/commit/de0e1646321acd9dd1547c3c6174db8f59540238 in it 🤔